common/src/main/java/io/netty/util/internal/PlatformDependent.java

```diff
@@ -104,6 +104,7 @@
     private static final ThreadLocalRandomProvider RANDOM_PROVIDER;
     private static final Cleaner CLEANER;
     private static final int UNINITIALIZED_ARRAY_ALLOCATION_THRESHOLD;
+    private static final Pattern BIT_PATTERN = Pattern.compile("([1-9][0-9]+)-?bit");
```

Comment:
why store this in memory? it's used once to init a field..

Reply:
that said I think we shouldn't use upper-case when declare it in the method and just use `bitPattern` or something like this

Reply:
Thanks for your advice.